SCClassLibrary/Common/GUI/PlusGUI/Control/ServerTreeView.sc

```diff
@@ -0,0 +1,204 @@
+ServerTreeView {
+	var <server, <bounds, viewParent, <window, resp, updateFunc, updater, tabSize = 25, treeViewStatus;
+
+	*new { |server, bounds, parent|
+		server  = server ?? { Server.default };
+		^super.newCopyArgs(server).makeWindow(bounds, parent)
+	}
+
+	makeWindow { |bounds, parent|
+		bounds = if(bounds.isNil) {
+			Rect(128, 64, 400, 400)
+		} {
+			var bnds = bounds.asRect;
+			if(bounds.asRect.width < 400) {
+				"The width value you set will be changed to 400, the minimum width.".postln;
+				Rect(bnds.left, bnds.top, 400, bnds.height);
+			} {
+				bounds
+			}
+		};
+		if(parent.isNil) {
+			window = Window(this.asString, bounds, scroll:true);
+		} {
+			window = parent
+		};
+		viewParent = window;
+		//if(window.view.respondsTo(\hasHorizontalScroller_)) { window.view.hasHorizontalScroller_(false) };
+		window.view.background_(Color.grey(0.9));
```

Comment:
the background should probably be in the view, not in the window. Maybe the UserView needs the background color?

All calls to the window that change its state should only happen if the window as locally created here. Otherwise, we want the responibility elsewhere. So `front` should be in line 22, I think.

Imagine you want to build a GUI and insert this view into it. Then suddenly your window comes to the front, which you may not have intended.